README.md

```diff
@@ -25,11 +25,13 @@ A Microverse exercise to implement the hamberger of the mobile version
 - [🙏 Acknowledgements](#acknowledgements)
 - [📝 License](#license)
 
-# 📖 Mobile-menu <a name="about-project"></a>
+# 📖 details popup window <a name="about-project"></a>
 
-> **Mobile-menu** is a starter exercise to create a porfolio capable to work in ever device, based on specific settings and requirements to ensure proper structure and best practices. By cloning this repo and running `npm install`  also with properly configured linters and correct initial file structrure. And also using the javascript to implement the hamberger of the mobile version.
+> **details popup window** is a starter exercise to create a porfolio capable to work in ever device, based on specific settings and requirements to ensure proper structure and best practices. By cloning this repo and running `npm install`  also with properly configured linters and correct initial file structrure. And also using the javascript to implement the hamberger of the mobile version.
 ## 🛠 Built With <a name="built-with"></a>
-
+-html 
+-css
+-javascript
 ### Tech Stack <a name="tech-stack"></a>
 
```

Comment:
- [ ] I think it would be better if you also update your `[Live Demo](https://Setupandmobilefirst.netlify.app/)` link as the current is not working, but broken instead.